core/src/main/java/org/apache/calcite/sql/SqlCloseCursor.java

```diff
@@ -0,0 +1,57 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to you under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.apache.calcite.sql;
+
+import org.apache.calcite.sql.parser.SqlParserPos;
+import org.apache.calcite.util.ImmutableNullableList;
+
+import java.util.List;
+import java.util.Objects;
+
+/**
+ * Parse tree for {@code SqlCloseCursor} call.
+ */
+public class SqlCloseCursor extends SqlCall {
+  private static final SqlSpecialOperator OPERATOR =
+      new SqlSpecialOperator("CLOSE", SqlKind.CLOSE);
+
+  public final SqlIdentifier cursorName;
+
+  /**
+   * Creates an instance of {@code SqlCloseCursor}.
+   *
+   * @param pos SQL parser position
+   * @param cursorName Name of the cursor to close, must not be null
+   */
+  public SqlCloseCursor(SqlParserPos pos, SqlIdentifier cursorName) {
+    super(pos);
+    this.cursorName = Objects.requireNonNull(cursorName);
+  }
+
+  @Override public void unparse(SqlWriter writer, int leftPrec, int rightPrec) {
```

Comment:
nit: Line wrap.

Reply:
In my editor it looks like this line is just at 80 chars.

Reply:
You're right - looks good.